src/interface/blas2/backend/power_vr.hpp

```diff
@@ -47,5 +47,19 @@ typename SB_Handle::event_t _gemv(SB_Handle& sb_handle, index_t _M, index_t _N,
 }
 }  // namespace backend
 }  // namespace gemv
+
```

Comment:
Please forgive me, if I misunderstood you, or I'm missing something.

Basically, I think we need a `_tbmv` definition within each `src/interface/blas2/backend/*.hpp` because they are mutual exclusively included by `src/interface/blas2/backend/backend.hpp`. This prevents us from having generic versions of a specific routine if a backend has it specialized.

Reply:
Yes that is fine, we have to redesign that in a way we could enable more backend